### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by ewnd9** [marcbachmann/limelight#1]
Hi. Do you plan to continue working on `limelight` or did you find a better alternative?

**Comment by marcbachmann** [comment]
Yes, but I don't have enough time. Would be nice to have some contributors.
No, I didn't find any alternative.

**Comment by ewnd9** [comment]
I learned about `limelight` in gitter chat of [`hain`](https://github.com/appetizermonster/hain) project, which is also an electron-based launcher. 

I like the extended preview of search results in html form in `limelight` so I'm interested in the project. I believe there is a great chance that both projects could share same plugins with some wrappers, so a possible merge is not necessary. 

https://github.com/ewnd9/limelight/commit/5c8b48c219af78b4a69ab075ee5aa139b9e0e1d4 in my fork contains changes to run on a Linux system too. I think I'm going to play with code for a while and then will contribute back.
